src/v0/destinations/dynamic_yield/transform.js

```diff
@@ -0,0 +1,110 @@
+const sha256 = require('sha256');
+const { EventType } = require('../../../constants');
+const { BASE_URL, ConfigCategory, mappingConfig } = require('./config');
+const {
+  defaultRequestConfig,
+  defaultPostRequestConfig,
+  simpleProcessRouterDest,
+  removeUndefinedAndNullAndEmptyValues,
+  constructPayload,
+} = require('../../util');
+const {
+  ConfigurationError,
+  TransformationError,
+  InstrumentationError,
+} = require('../../util/errorTypes');
+const { preparePayload } = require('./util');
+
+const responseBuilder = (payload, apiKey, endpoint) => {
+  if (payload) {
+    const response = defaultRequestConfig();
+    response.endpoint = endpoint;
+    response.headers = {
+      'Content-Type': 'application/json',
+      'DY-API-Key': apiKey,
+    };
+    response.method = defaultPostRequestConfig.requestMethod;
+    response.body.JSON = removeUndefinedAndNullAndEmptyValues(payload);
+    return response;
+  }
+  throw new TransformationError('Payload could not be populated due to wrong input');
+};
+
+const identifyResponseBuilder = (message, { Config }) => {
+  const { apiKey, hashEmail } = Config;
+
+  const payload = constructPayload(message, mappingConfig[ConfigCategory.IDENTIFY.name]);
```

Comment:
Move the below code L37-L56 to `util.js` under `prepareIdentifyPayload` and add unit test to it

Reply:
Done